### PR TITLE
QuangVu: commit exam1

### PR DESCRIPTION
**Pull request by Quangsumi**

### CODE REVIEW

File: QuangVu/exam1/exam1.css [hunk 1]
```diff
@@ -0,0 +1,150 @@
+* {padding: 0px; margin: 0px;}
+
+body {
+    background-color: rgb(245, 244, 244);
```
Comment (dinhhung09138) [inline QuangVu/exam1/exam1.css:4]: Please use hex color code.

**Review by dinhhung09138 — COMMENTED** [review]
Why do you delete file gitignore.
Please reset it.

File: QuangVu/exam1/exam1.css (hunk 1 above)
Comment (Quangsumi) [reply QuangVu/exam1/exam1.css:4]: I've added .gitignore back. Check it again, please

Why can't we use RBG color????

**Review by dinhhung09138 — APPROVED** [review]
Done

**Pull request merged**